data/json/monsters/rodentkin.json

```diff
@@ -141,6 +141,45 @@
     "flags": [ "SEES", "SMELLS", "HEARS", "WARM", "SWIMS", "PATH_AVOID_DANGER_1", "CAN_DIG" ],
     "dissect": "dissect_rat_sample_small"
   },
+  {
+    "id": "mon_pack_rat",
+    "type": "MONSTER",
+    "name": { "str": "pack rat" },
+    "description": "This humanoid rodent stands over five feet tall, it pokes and bites at anything shiny, hastily shoving anything of interest into its mouth",
```

Comment:
Keeping stuff in their cheeks is a hamster thing, not a rat thing as far as I know.

Reply:
That's true. Normal IRL pack rats do carry single objects in their hands or mouth to transport back to their nests though, unless I'm misinformed. I thought it wouldn't be too much of a stretch to imagine that a significantly larger packrat could carry more than one coin/piece of trash.